src/FeaturePlugin.php

```diff
@@ -148,7 +148,7 @@ protected function define_constants() {
 		$this->define( 'WC_ADMIN_PLUGIN_FILE', WC_ADMIN_ABSPATH . 'woocommerce-admin.php' );
 		// WARNING: Do not directly edit this version number constant.
 		// It is updated as part of the prebuild process from the package.json value.
-		$this->define( 'WC_ADMIN_VERSION_NUMBER', '0.24.0' );
+		$this->define( 'WC_ADMIN_VERSION_NUMBER', '0.25.1' );
```

Comment:
You can omit this change. This and several other places get updated by `npm run pre-release`. This is why tests are failing. Perhaps just update the test instructions instead?